pcsx2/Elfheader.cpp

```diff
@@ -149,7 +149,7 @@ void ElfObject::initElfHeaders()
 
 bool ElfObject::hasValidPSXHeader()
 {
-	if (data.GetSizeInBytes() < sizeof(PSXEXEHeader))
+	if (static_cast<u64>(data.GetSizeInBytes()) < sizeof(PSXEXEHeader))
```

Comment:
I wouild sign the PSXEXEHeader, tbh.  just in case GetSizeInBytes returns -1 if it can't find the file or whatever, you just made it U64_MAX which would pass this check.